omod/src/main/webapp/resources/scripts/jquery-ui/js/openmrsSearch.js

```diff
@@ -1075,14 +1075,16 @@ function OpenmrsSearch(div, showIncludeVoided, searchHandler, selectionHandler,
             return Math.ceil(this._table.fnSettings()._iDisplayStart / this._table.fnSettings()._iDisplayLength) + 1;
         },
         _updatePageInfo: function(searchText) {
-            textToDisplay = omsgs.viewingResultsFor.replace("_SEARCH_TEXT_", "'<b>"+searchText+"</b>'");
+            if ((/^[a-zA-Z0-9_\-]+$/.test(searchText))) {
+            textToDisplay = omsgs.viewingResultsFor.replace("_SEARCH_TEXT_", "'<b>"+sanitizeHtml(searchText)+"</b>'");
             if($j.trim(searchText) == '')
                 textToDisplay = omsgs.viewingAll;
 
             $j('#pageInfo').html(sanitizeHtml(textToDisplay));
 
             if($j('#pageInfo').css("visibility") != 'visible')
                 $j('#pageInfo').css("visibility", "visible");
```

Comment:
The indentation is off. Could you please adjust?

Reply:
@rkorytkowski  resolved